src/app/core/services/auth.service.ts

```diff
@@ -60,15 +60,19 @@ export class AuthService {
     this.phaseService.reset();
     this.dataService.reset();
     this.githubEventService.reset();
-    this.titleService.setTitle(
-      require('../../../../package.json').name
-      .concat(' ')
-      .concat(require('../../../../package.json').version)
-    );
+    this.setTitle();
     this.issueService.setIssueTeamFilter('All Teams');
     this.reset();
   }
 
+  setTitle(): void {
+    const appSetting = require('../../../../package.json');
+    const title = appSetting.name
+      .concat(' ')
+      .concat(appSetting.version);
```

Comment:
Similar to the comments above perhaps we could use string interpolation here too?